app/src/main/java/org/mozilla/fenix/tabtray/TabTrayView.kt

```diff
@@ -245,6 +246,8 @@ class TabTrayView(
                 View.VISIBLE
             }
             view.tab_tray_overflow.isVisible = !hasNoTabs
+
+            counter_text.text = "${state.normalTabs.size}"
```

Comment:
Can you file a followup to dynamically change the content description to match the num of tabs open?

Reply:
Ah, I can just quickly do that now.